cmd/controller/main.go

```diff
@@ -108,10 +112,20 @@ func main() {
 		log.Fatal(http.ListenAndServe(":"+port, mux))
 	}()
 
+	taskrunControllerConfig := taskrun.ControllerConfiguration{
+		Images:                   images,
+		DisableTaskRefResolution: *experimentalDisableInTreeResolution,
+	}
+
+	pipelinerunControllerConfig := pipelinerun.ControllerConfiguration{
+		Images:                       images,
+		DisablePipelineRefResolution: *experimentalDisableInTreeResolution,
+	}
```

Comment:
Seems inconsistent with `pipeline.Images` for these to specialize between the two controllers, since the pipeline run controller only uses a subset of the `pipeline.Images` struct, but the same "options" struct is passed to both controllers.

My personal preference would be to rename/generalize the `Images` struct to reflect a more complete set of options, but this seems nitpicky and I have some other changes I've been thinking about to some of this handling, so perhaps we land this and I can try to find something we're happy with.

thanks for the change. 👍